documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,29 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (All optional):
+
+* *Preserving the status ID of Windows VMs*: By default, Virtual NICs (VNICs) change during the migration process, which results in Windows VNICs that are set with static IP in vSphere to lose their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, toggle the *Whether to preserve the static IPs* switch in the window that opens, and then click *Save*.
+
+* *Entering a list of decryption passphrases for disks encrypted using Linux Unified Key Setup (LUKS)*: To enter a list of decryption passphrases LUKS-encrypted devices, in the *Settings* section, click the Edit icon next to *Disk decryption passphrases*, enter the passphrases, and then click *Save*. You do not need to enter the passphrases in a specific order -- for each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.
+
+* *Specifying a root device*: Applies to multi-boot VM migrations only. By default, {project-short} uses the first bootable device detected as the root device.
++
+To specify a different root device, in the *Settings* section, click the Edit icon next to *Root device* and choose a device from the list of commonly-used options or enter a device in the text box.
++
+{project-short} uses the following format for disk location: `/dev/sdX#`, where X represents the disk number in letters (that is the first disk would be represented as 'a', the second disk as 'b', and so on) and # represents the partition that holds the operating system inside the disk (starting from 1). For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
```

Comment:
```suggestion
{project-short} uses the following format for disk location: `/dev/sdX#`, where X represents the disk number in letters (so that the first disk would be represented as 'a', the second disk as 'b', and so on) and # represents the partition that holds the operating system inside the disk (starting from 1). For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
```

Reply:
Following Bella's suggestion, I changed the description of the format, eliminating the need for the parenthetical comment. "So that" is a better phrase than "that is," so thanks for the note.